common/djangoapps/student/cookies.py

```diff
@@ -5,6 +5,8 @@
 import time
 import json
 
+from email_marketing.cookies import add_email_marketing_cookies
```

Comment:
I could switch to using a signal.  Signal receivers can return a response, in this case it could be a cookie that student/cookies than adds to the http response